README.md

```diff
@@ -0,0 +1,176 @@
+![About Openmeetings Logo](/openmeetings-server/src/site/resources/images/logo.png)
```

Comment:
Maybe we can use README.md, and remove README?

Reply:
I think we can, also I can rewrite other information text files into markdown format.

Reply:
Great, which files are you going to change?

Reply:
I think about CHANGELOG and NOTICE files.

Reply:
Let's do it, I found no restrictions on format of these files

Reply:
Done!